app/src/main/java/com/shobhith/atvproductsapp/home/domain/use_case/GetProductByCategoryName.kt

```diff
@@ -0,0 +1,8 @@
+package com.shobhith.atvproductsapp.home.domain.use_case
```

Comment:
let the package name be simple without any underscore. like usecase. use_case not required

Reply:
Fixed.